subnet/deployment-generator/src/gen.js

```diff
@@ -19,9 +19,9 @@ Object.freeze(config)
 // const output_path = `${__dirname}/../generated/`
 
 
-keys = gen_other.genSubnetKeys()  
+let keys = gen_other.genSubnetKeys()  
```

Comment:
for better JS coding convention, try to use eslint. It will review all your coding style.

Reply:
thank you. let me check.